sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2146,4 +2146,17 @@ public void testRefreshIgnoresClosedPartitions() {
             is("1| 1\n")
         );
     }
+
+    @Test
+    public void testWhereOnPartitionedAndOtherColumn() {
```

Comment:
maybe rename to something like `test_partition_filter_and_column_filter_are_both_applied`